.travis.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+case "$1" in
+    server)
+        cd server
+        bundle install
+        bundle exec rspec spec
```

Comment:
```suggestion
        bundle exec rspec
```

RSpec's default directory is `spec`, so it may be easier to not mention it.

Reply:
The above is totally not a blocker to merging this!